images/win/scripts/Installers/Install-Firefox.ps1

```diff
@@ -4,17 +4,66 @@
 ################################################################################
 
 Import-Module -Name ImageHelpers -Force
-
 $temp_install_dir = 'C:\Windows\Installer'
 New-Item -Path $temp_install_dir -ItemType Directory -Force
 
-Install-EXE -Url "https://seleniumwebdrivers.blob.core.windows.net/knownfirefoxversion/FirefoxSetup.exe" -Name "FirefoxSetup.exe" -ArgumentList "-ms"
+$versionsJson = Invoke-WebRequest  "https://product-details.mozilla.org/1.0/firefox_versions.json" | ConvertFrom-Json
+$latestVersion = $($versionsJson.LATEST_FIREFOX_VERSION)
+Write-Host "Firefox latest version: $latestVersion"
+
+# url for latest version of firefox
+$urlLatestVersion = "https://download.mozilla.org/?product=firefox-latest&os=win64&lang=en-US"
+Install-EXE -Url $urlLatestVersion -Name "Firefox Setup $latestVersion.exe" -ArgumentList ("/silent", "/install")
 
-$path = '{0}\Program Files\Mozilla Firefox\' -f $env:SystemDrive;
-New-Item -path $path -Name 'mozilla.cfg' -Value '//
+# Disable autoupdate
+$firefoxDirectoryPath = '{0}\Program Files\Mozilla Firefox\' -f $env:SystemDrive;
+New-Item -path $firefoxDirectoryPath -Name 'mozilla.cfg' -Value '//
 pref("browser.shell.checkDefaultBrowser", false);
 pref("app.update.enabled", false);' -ItemType file -force
 
-$path = '{0}\Program Files\Mozilla Firefox\defaults\pref\' -f $env:SystemDrive;
-New-Item -path $path -Name 'local-settings.js' -Value 'pref("general.config.obscure_value", 0);
+$firefoxPreferencesFolder = Join-Path $firefoxDirectoryPath "defaults\pref"
+New-Item -path $firefoxPreferencesFolder -Name 'local-settings.js' -Value 'pref("general.config.obscure_value", 0);
 pref("general.config.filename", "mozilla.cfg");' -ItemType file -force
+
+Write-Host "Install Firefox WebDriver"
+$DestinationPath = "$($env:SystemDrive)\";
```

Comment:
This variable is used only in one place (line 40). probably we can remove it and just write: `$FirefoxDriverPath = Join-Path $env:SystemDrive "SeleniumWebDrivers\GeckoDriver"`

Reply:
Done.